src/Microsoft.Graph/Models/Generated/FileAttachment.cs

```diff
@@ -30,7 +30,7 @@ public partial class FileAttachment : Attachment
     
         /// <summary>
         /// Gets or sets content location.
-        /// The Uniform Resource Identifier (URI) that corresponds to the location of the content of the attachment.
```

Comment:
@MIchaelMainer , @darrelmiller Note this change.

Reply:
Documentation is the one area where changes can be destructive with less concern.